api/models/languages_levels.py

```diff
@@ -25,9 +25,18 @@ class TeachingLanguage(MultilingualModel):
 
 class LanguageLevel(models.Model):
     name = models.CharField(max_length=3, unique=True)
-    rank = models.IntegerField(unique=True)
+    rank = models.PositiveSmallIntegerField(unique=True)
+
+    def __str__(self):
+        return self.name
 
 
 class TeachingLanguageAndLevel(models.Model):
     language = models.ForeignKey(TeachingLanguage, on_delete=models.CASCADE)
```

Comment:
`TeachingLevel` is only a level, without a language code. And its name is fixed and explicit: "A0", "A1", "A2" etc. Will be pre-populated as well. Or actually we'll turn it into an enum because levels are international and will probably never(-ish) change,

Reply:
Let's discuss enums in general additionally

Reply:
👍🏻